node/src/cli.rs

```diff
@@ -127,8 +148,8 @@ pub struct RunCmd {
 
     /// Listen address in the P2P network of Storage Providers and Collators
     /// that the bootstrap node binds to.
-    #[arg(long, required = false)]
-    pub p2p_listen_address: Option<Multiaddr>,
+    #[arg(long, default_value_t=default_p2p_tcp_multiaddr(), value_parser = validate_tcp_multiaddr)]
```

Comment:
```suggestion
    #[arg(long, default_value_t = default_p2p_tcp_multiaddr(), value_parser = validate_tcp_multiaddr)]
```
Formatting to keep it consistent with the other `default_value_t`'s